src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java

```diff
@@ -658,10 +669,23 @@ private PaymentProvider buildPaymentProvider(final AccountData account, @Nullabl
         paymentProvider.setCountryIsoCode(paymentProviderCountryIsoCode);
         paymentProvider.setPaymentType(paymentProviderPaymentType);
         paymentProvider.setRecurringType(paymentProviderRecurringType);
+        paymentProvider.setThreeDThreshold(parseThreeDThreshold(threeDThreshold));
```

Comment:
paymentprovider.threeDThresholdOriginal is in fact an Integer. So i would propose to change the setter also to Int (we cant delete the conversion logic there) and also directly use Int instead of Long here.